api-flask/requirements.txt

```diff
@@ -1,6 +1,8 @@
 Flask-Caching==1.9.0
 Flask-Cors==3.0.9
 flask-restx==0.5.1
+fastapi==0.75.1
```

Comment:
is this not already installed by the docker?

Reply:
Let's limit the requirements.txt to what we actually need to add on top of `tiangolo/uvicorn-gunicorn-fastapi:python3.8`